index.js

```diff
@@ -0,0 +1,45 @@
+let image = document.querySelector(".profile-pic");
+
+image.onclick = function () {
+  alert("Not available!");
+}
+
+let myHeaders = new Headers();
+myHeaders.append("Origin", "https://localhost.com");
+
+let requestOptions = {
+  method: 'GET',
+  headers: myHeaders,
+  redirect: 'follow'
+};
+let gallery = document.querySelector(".picture-grid");
+
+
+let arrayPromise = fetch("https://image-gallery-pjks.s3.ca-central-1.amazonaws.com/data.json", requestOptions)
+  .then(res => res.json())
+  .then(res => {
+    console.log(res)
+    return res;
+  });
+
```

Comment:
Also I think it makes sense to handle errors with `catch()` just incase something goes down but thats optional just wanted to point it out

Reply:
its done in another PR, wanted him to learn to split work up, so take a look at that one